update.py

```diff
@@ -63,7 +64,9 @@ def check_notices(session, sessionData):
             notice['attachment_url'] = ERP_ATTACHMENT_URL + m.group(1)
             r = session.get(notice['attachment_url'], stream=True)
             r.raw.decode_content = True
+            notice['attachment_raw'] = r
```

Comment:
This actually adds the request object itself rather than the raw data. What is the type of `request.raw`, and can we use that directly?

Reply:
`request.raw` is a `urllib3.response.HTTPResponse` object, and we can use it directly too. 👍